App/StackExchange.DataExplorer.Tests/packages.config

```diff
@@ -3,5 +3,5 @@
   <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net451" />
   <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net451" />
   <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net451" />
-  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net451" />
+  <package id="Newtonsoft.Json" version="10.0.3" targetFramework="net48" />
```

Comment:
Would eliminate packages.config and move to `<PackageReference>` - without any packages checked in!

Reply:
👍 done!